rss-client/src/app/Quiz/employee/invalid-quiz/invalid-quiz.component.ts

```diff
@@ -15,11 +15,20 @@ export class InvalidQuizComponent implements OnInit {
   
   ngOnInit(): void {
     this.cheaterService.invalidated.subscribe(e => {
-      
+      // if (e) {
+      //   if (!this.open)
+      //   this.openDialog()
+      // }
     })
   }
 
-  
+  // openDialog() {
+  //   const dialogRef = this.dialog.open(InvalidQuizDialogContent);
+  //   this.open = true
+  //   dialogRef.close().subscribe(() => {
+  //     this.open = false
+  //   });
+  // }
 
```

Comment:
why are we committing commented code?

Reply:
This is not us1 code. We did not work in this module and don't feel comfortable taking it out. We did pull from userstory2.